scripts/add-chain.sh

```diff
@@ -75,6 +75,11 @@ genesis:
     hash: "$(jq -j .genesis.l2.hash $ROLLUP_CONFIG)"
     number: $(jq -j .genesis.l2.number $ROLLUP_CONFIG)
   l2_time: $(jq -j .genesis.l2_time $ROLLUP_CONFIG)
+
+canyon_time: $(jq -j .canyon_time $ROLLUP_CONFIG)
+delta_time: $(jq -j .delta_time $ROLLUP_CONFIG)
+ecotone_time: $(jq -j .ecotone_time $ROLLUP_CONFIG)
```

Comment:
Desiderata for improvements here:
* we pipe the values through `date -r <timestamp> -u ` and append this as a comment on each line, so it matches: 
https://github.com/ethereum-optimism/superchain-registry/blob/119f493760a5d2de4c9cb878dcfa03077e24d2bc/superchain/configs/sepolia/superchain.yaml#L10-L12
* we iterate through all fields ending in `_time`, to catch future hardforks without needing to update the script
* instead of filling with `null` when the `jq` query fails, we should use `nil` or even better just omit the entire line from the output yaml file. 

IMO we tackle these in a followup issue / PRs. This PR is reasonably urgent, because it will cause tests to incorrectly fail on in flight standard chain PRs.

Reply:
Works for me, let's just remember to track this follow up

Reply:
#166